SpaceDuck.UserService/SpaceDuck.UserService/Models/LoginModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SpaceDuck.UserService.Models
+{
+    public class LoginModel
+    {
+        public string Email { get; set; }
```

Comment:
Logujemy się loginem, nie mailem

Reply:
Ew. UserName

Reply:
to jest już zadecydowane? Bo imo najlepiej by było jakby można było tym lub tym (to chyba teraz standard żeby można się było zalogować też mailem)

Reply:
no mi też się wydaje że mail jest częstszy ale mogę dodać model z userName

Reply:
Okej, to zróbmy możliwość tym i tym.